src/status_im/extensions/ethereum.cljs

```diff
@@ -98,8 +98,119 @@
+
+(handlers/register-handler-fx
+ :extensions/ethereum-erc20-transfer
+ (fn [{db :db} [_ _ {:keys [address to value] :as arguments}]]
+   (let [arguments' (merge arguments
+                           {:to address
+                            :method "transfer(address,uint256)"
+                            :params [to value]
+                            :value nil})]
+     (wrap-with-resolution db arguments' :to execute-send-transaction))))
+
+(handlers/register-handler-fx
+ :extensions/ethereum-erc20-transfer-from
+ (fn [{db :db} [_ _ {:keys [address from to value] :as arguments}]]
+   (let [arguments' (merge arguments
+                           {:to address
+                            :method "transferFrom(address,address,uint256)"
+                            :params [from to value]
+                            :value nil})]
+     (wrap-with-resolution db arguments' :to execute-send-transaction))))
+
+(handlers/register-handler-fx
+ :extensions/ethereum-erc20-approve
+ (fn [{db :db} [_ _ {:keys [address spender value] :as arguments}]]
+   (let [arguments' (merge arguments
+                           {:to address
+                            :method "approve(address,uint256)"
+                            :params [spender value]
+                            :value nil})]
+     (wrap-with-resolution db arguments' :to execute-send-transaction))))
+
+(handlers/register-handler-fx
+ :extensions/ethereum-erc721-owner-of
+ (fn [{db :db} [_ _ {:keys [address token-id] :as arguments}]]
+   (let [arguments' (merge arguments
+                           {:address address
+                            :method "ownerOf(uint256)"
+                            :params [token-id]
+                            :outputs ["address"]})]
+     (wrap-with-resolution db arguments' :address execute-ethcall))))
+
+(handlers/register-handler-fx
+ :extensions/ethereum-erc721-is-approved-for-all
+ (fn [{db :db} [_ _ {:keys [address owner operator] :as arguments}]]
+   (let [arguments' (merge arguments
+                           {:address address
+                            :method "isApprovedForAll(address,address)"
+                            :params [owner operator]
+                            :outputs ["bool"]})]
+     (wrap-with-resolution db arguments' :address execute-ethcall))))
+
+(handlers/register-handler-fx
+ :extensions/ethereum-erc721-get-approved
+ (fn [{db :db} [_ _ {:keys [address token-id] :as arguments}]]
+   (let [arguments' (merge arguments
+                           {:address address
+                            :method "getApproved(uint256)"
+                            :params [token-id]
+                            :outputs ["address"]})]
+     (wrap-with-resolution db arguments' :address execute-ethcall))))
+
+(handlers/register-handler-fx
+ :extensions/ethereum-erc721-set-approval-for-all
+ (fn [{db :db} [_ _ {:keys [address to approved] :as arguments}]]
+   (let [arguments' (merge arguments
+                           {:to address
```

Comment:
What is the rational of `to` vs `address` ?

Reply:
In `execute-send-transaction` didn't changed the `:to` into `:address` like I did with `eth_call`. if it is ok for you i'll change all, so `ethereum/send-transaction` parameters will change too, ok?

Reply:
Why do we want one vs the other?
Ultimately if it's the same info, should be the same property name?
Also we might want to stick with what JSON-RPC uses

Reply:
```
(handlers/register-handler-fx
 :extensions/ethereum-erc721-set-approval-for-all
 (fn [{db :db} [_ _ {:keys [address to approved] :as arguments}]]
   (let [json-rpc-args {:to address
                                     :method "setApprovalForAll(address,bool)"
                                     :params [to approved]}]
     (wrap-with-resolution db json-rpc-args :to execute-send-transaction))))
```